### PR TITLE
feat: add sensing launch

### PR DESCRIPTION
**Pull request by 1222-takeshi**
## Related Issue(required)
#1 
## Description(required)
add sensing_launch package
## Review Procedure(required)
``` sh
$ ros2 launch sensing_launch sensing.launch.xml sensor_model:=xx1
```
(For launching the simulator, you needs the aip_xx1_description package)
## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Review by taikitanaka3 — APPROVED** [review]

File: launch/sensing_launch/launch/sensing.launch.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<launch>
+
+  <arg name="launch_driver" default="true" description="do launch driver"/>
+  <arg name="sensor_model" default="aip_xx1" description="sensor model name"/>
+  <arg name="vehicle_param_file" default="$(find-pkg-share vehicle_info_util)/config/vehicle_info.param.yaml" description="path to the file of vehicle info yaml"/>
```
Comment (taikitanaka3) [inline launch/sensing_launch/launch/sensing.launch.xml:5]: @1222-takeshi 
It's necessary to add vehicle_info_util package as exec depend

File: launch/sensing_launch/package.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <version>0.1.0</version>
+  <description>The sensing_launch package</description>
+
+  <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <exec_depend>aip_x1_launch</exec_depend>
+  <exec_depend>aip_x2_launch</exec_depend>
+  <exec_depend>aip_xx1_launch</exec_depend>
```
Comment (taikitanaka3) [inline launch/sensing_launch/package.xml:14]: @1222-takeshi 
aip_xx1_launch uses `velodyne_point_cloud` pkg so you need to add build_depends.repos

Comment (taikitanaka3) [reply launch/sensing_launch/package.xml:14]: Here is a repo https://github.com/tier4/velodyne_vls

File: launch/sensing_launch/launch/sensing.launch.xml (hunk 1 above)
Comment (1222-takeshi) [reply launch/sensing_launch/launch/sensing.launch.xml:5]: fixed https://github.com/autowarefoundation/autoware.universe/pull/165/commits/9fe36debcfd418f211425bc7b184617b0027a00f

File: launch/sensing_launch/package.xml (hunk 2 above)
Comment (1222-takeshi) [reply launch/sensing_launch/package.xml:14]: fixed https://github.com/autowarefoundation/autoware.universe/pull/165/commits/0708b9fc8da727f943849dfc7877f6b4f993fc90

**Review by taikitanaka3 — APPROVED** [review]

**Comment by taikitanaka3** [comment]
@1222-takeshi 
sensing_launch: Cannot locate rosdep definition for [common_sensor_launch]
common_sensor_launch package is necessary

**Pull request merged**